thorlcr/graph/thgraph.cpp

```diff
@@ -118,7 +126,9 @@ class CThorGraphResult : implements IThorResult, implements IRowWriter, public C
     }
     virtual IThorRowInterfaces *queryRowInterfaces() { return rowIf; }
     virtual CActivityBase *queryActivity() { return &activity; }
-    virtual bool isDistributed() const { return distributed; }
+    virtual bool isDistributed() const { return ((unsigned)resultType) & thorgraphresult_distributed; }
+    virtual bool isSparse() const { return ((unsigned)resultType) & thorgraphresult_sparse; }
```

Comment:
See comment on the enumeration below - this will return true if the results is grouped (and grouped will return true if it is sparse).